src/lib/pubkey/curve448/curve448_scalar.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Ed448 Scalar
+ * (C) 2024 Jack Lloyd
+ *     2024 Fabian Albert - Rohde & Schwarz Cybersecurity
+ *
+ * Botan is released under the Simplified BSD License (see license.txt)
+ */
+#ifndef BOTAN_CURVE448_SCALAR_H_
+#define BOTAN_CURVE448_SCALAR_H_
+
+#include <botan/strong_type.h>
+#include <botan/types.h>
+#include <botan/internal/bit_ops.h>
+#include <botan/internal/loadstor.h>
+
+namespace Botan {
+
+constexpr size_t words_for_bits(size_t x) {
+   constexpr size_t word_bits = sizeof(word) * 8;
+   return (x + word_bits - 1) / word_bits;
+}
+
+constexpr size_t WORDS_446 = words_for_bits(446);
+constexpr size_t BYTES_446 = ceil_tobytes(446);
```

Comment:
I would prefer these as constants associated with the struct, eg `Scalar448::BYTES`